public/js/monitor.js

```diff
@@ -548,14 +531,10 @@ function vpnBannerLogic(options) {
   }
 
   // Only show banner if users first language is some English-locale variant
```

Comment:
I think this comment is no longer applicable?

Reply:
Removing!